mozilla_aws_cli/utils.py

```diff
@@ -30,7 +30,7 @@ def generate_challenge(code_verifier):
         hashlib.sha256(code_verifier.encode()).digest())
 
 
-def role_arn_to_profile_name(role_arn, role_map):
+def role_arn_to_role_name(role_arn, role_map):
```

Comment:
If this gets renamed maybe call it something other than `role_arn_to_role_name` as the value this produces isn't the role name, it's the union of the account name and role name.

Maybe `role_arn_to_display_name` or something

Reply:
Done!